wiki/Beatmap_Editor/Beat_Snap_Divisor/de.md

```diff
@@ -25,14 +65,60 @@ Weiße Linie: Viervierteltakt
-| Beat               | Volle-Beats                                       | Halbe-Beats                                              | Tripel                                        | Viertel-Beats                                            |
-| Tick Noten         | Nur weiße Ticks                                   | Fügt rote Ticks hinzu                                    | Hat zwei lila Ticks zwischen jeden roten Tick | Fügt gelbe Ticks als Ergänzung zu roten und blauen hinzu |
-| Noten              | Am besten für Easy Schwierigkeitsstufen geeignet. |                                                          | Compound Timing                               | Nützlich für generelles Mapping                          |
-| Halber Grundschlag | ![](1_6_m.jpg "1_6_m.jpg")                        | ![](1_8_m.jpg "1_8_m.jpg")                               | ![](1_12_m.jpg "1_12_m.jpg")                  | ![](1_16_m.jpg "1_16_m.jpg")                             |
-| Divisor            | 1/6                                               | 1/8                                                      | 1/12                                          | 1/16                                                     |
-| Beat               | Doppelte-Tripel                                   | 8. Beats                                                 | 12. Beats                                     | 16. Beats                                                |
-| Tick Noten         | Hat zwei lila Ticks zwischen jeden roten Tick     | Fügt gelbe Ticks als Ergänzung zu roten und blauen hinzu |                                               |                                                          |
-| Noten              | Compound timing                                   |                                                          |                                               |                                                          |
+### Divisor 1/1
+
+![](/wiki/shared/1_1_m.jpg)
+
+- Volle-Beats
+- Nur weiße Ticks
+- Am besten für Easy Schwierigkeitsstufen geeignet.
+
+### Divisor 1/2
+
+![](/wiki/shared/1_2_m.jpg)
+
+- Halbe-Beats
+- Fügt rote Ticks hinzu
+
+### Divisor 1/3
+
+![](/wiki/shared/1_3_m.jpg)
+
+- Tripel
+- Hat zwei lila Ticks zwischen jeden roten Tick
+- Compound Timing
+
+### Divisor 1/4
+
+![](/wiki/shared/1_4_m.jpg)
+
+- Viertel-Beats
+- Fügt gelbe Ticks als Ergänzung zu roten und blauen hinzu
+- Nützlich für generelles Mapping
+
+### Divisor 1/6
+
+![](/wiki/shared/1_6_m.jpg)
+
+- Doppelte-Tripel
+- Hat zwei lila Ticks zwischen jeden roten Tick
+- Compound timing
+
+### Divisor 1/8
+
+![](/wiki/shared/1_8_m.jpg)
+
+- 8\. Beats
+- Fügt gelbe Ticks als Ergänzung zu roten und blauen hinzu
+
+### Divisor 1/12
+
+![](/wiki/shared/1_12_m.jpg)
+
+-	12\. Beats
+
+### Divisor 1/16
+
+![](/wiki/shared/1_16_m.jpg)
```

Comment:
L122: Add text on ``[]`` to phrase which supposed to be there if the image doesn't load